src/formGenerator.vue

```diff
@@ -301,10 +310,16 @@ div.vue-form-generator(v-if='schema != null')
 			// Validating the model properties
 			validate() {
 				this.clearValidationErrors();
+				this.touched.splice(0);
 
 				this.$children.forEach((child) => {
 					if (isFunction(child.validate))
 					{
+						this.touched.push({
```

Comment:
In fact, I started with this strategy (to have touched inside field data), but I didn't think about doing what you suggest.
The problem is that the style is decided `getFieldRowClasses` (in `formGenerator.vue`). This method argument is the field definition from the schema. This field definition is what is used to match "local" errors with "global" errors (`fieldErrors` for example).
I don't know why @icebob used this way to match things, but I can assume that there is a reason for it, a consequence I don't foresee right now (maybe for groups ?).
Anyway, I'm going to try another way to match fields for `touched` property.